docs/TESTS.md

```diff
@@ -1,15 +1,20 @@
 # Tests
 
-<!-- TODO: write document
+To run the tests for your SQLite exercises, follow these steps:
 
-  This document should describe everything related to running tests in the track.
+1. Open your preferred terminal.
+2. Navigate to the directory containing the appropriate `STUB_test.sql` file, where `STUB` is the name of the exercise, using hyphens instead of spaces and all lowercase.
+3. Execute the following command: 
+```bash
+sqlite3 '' --init STUB_test.sql -bail '.exit'
+```
+  
+This command initializes SQLite with an empty database, runs the tests specified in STUB_test.sql, and exits afterward.
+`-bail` will make sure the code execution stops for invalid code.
 
-  If your track uses skipped tests, this document can explain why thet is used and
-  how to unskip tests.
+The test results will be displayed in the console as a table, showing the description of each test, its status (e.g., "passed"), and a message if the status is not "passed."
 
-  This document can also link to the testing framework documentation.
+## Execution Details
 
-  The contents of this document are displayed on the track's documentation
-  page at `https://exercism.org/docs/tracks/<track>/tests`.
-
-  See https://exercism.org/docs/building/tracks/docs for more information. -->
+For those interested in understanding the test file (STUB_test.sql), it prepares the initial database by populating it with values from `data.csv` for valid data and `error_data.csv` for entries that should fail.
```

Comment:
```suggestion
The test file (STUB_test.sql) is a pretty complex script.
Here is an overview of what it does.

1. Sets up tables by reading `create_fixture.sql`.
   This creates tables then populates them with values from `data.csv` for valid data and `error_data.csv` for entries that should fail.
2. Runs the solution code by reading the `${slug}.sql` file.
3. Creates tables to hold test cases and results.
4. Populates the test results based on the data in the table(s) updated by the solution.
5. Displays test results (pass or fail and optional details).
```

Reply:
In step 1 we could be explicit and mention that the solution is expected to work with these tables (only).